dvc/cache/base.py

```diff
@@ -101,24 +101,26 @@ def changed(self, path_info, hash_info):
         Args:
             path_info: dict with path information.
             hash: expected hash value for this data.
+            filter_info: an optional argument to target a specific file.
 
         Returns:
             bool: True if data has changed, False otherwise.
         """
 
+        path = filter_info or path_info
         logger.trace(
-            "checking if '%s'('%s') has changed.", path_info, hash_info
+            "checking if '%s' ('%s') has changed.", path_info, hash_info
```

Comment:
Minor: this is an unrelated change. This should either be a separate commit or should not be in this PR at all. 
```suggestion
            "checking if '%s'('%s') has changed.", path_info, hash_info
```